Index: app.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import pygame\r\ndef app():\r\n\r\n\r\n\r\nif __name__ == \"__app__\":\r\n    app()
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app.py b/app.py
--- a/app.py	(revision 947be342620f3c74aa96a4fb04fbc82624607a87)
+++ b/app.py	(date 1705512279214)
@@ -1,7 +1,9 @@
 import pygame
+
+
 def app():
-
+    pass
 
 
 if __name__ == "__app__":
-    app()
\ No newline at end of file
+    app()
